### PR TITLE
Uniform release naming

### PR DESCRIPTION
**Pull request by atomgomba**
Screenshot from 2023-10-20 14-18-53

### CODE REVIEW

File: docs/release/Betaflight-3.0.x-Release-Notes.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# BetaFlight 3.0 Release Notes
+# BetaFlight 3.0.x Release Notes
```
Comment (haslinghuis) [inline docs/release/Betaflight-3.0.x-Release-Notes.md:2]: Imho we should not include patch version (only major.minor)

Comment (atomgomba) [reply docs/release/Betaflight-3.0.x-Release-Notes.md]: Okay, fixed :+1:

File: docs/release/Betaflight-3.1.x-Release-Notes.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+
+# BetaFlight 3.1.x Release Notes
```
Comment (haslinghuis) [inline docs/release/Betaflight-3.1.x-Release-Notes.md:2]: Same here :)

**Review by nerdCopter — DISMISSED** [review]

File: docs/release/Betaflight-3.0-Release-Notes.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # BetaFlight 3.0 Release Notes
 
 This is really a major release. The full detailed change list can be found in the the commit history.
-[https://github.com/betaflight/betaflight/commits/master]
+[https://github.com/betaflight/betaflight/commits/master]
```
Comment (haslinghuis) [inline docs/release/Betaflight-3.0-Release-Notes.md:5]: 👾

Comment (nerdCopter) [reply docs/release/Betaflight-3.0-Release-Notes.md:5]: good catch, trailing space.  most editors can auto-fix this if configured. (configure for new/edited lines only, not untouched lines)

Comment (atomgomba) [reply docs/release/Betaflight-3.0-Release-Notes.md:5]: at this point I'm going to throw VS Code out of the window(s)

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**